packages/puppeteer-crawler/src/internals/puppeteer-crawler.ts

```diff
@@ -4,13 +4,15 @@ import {
     BrowserCrawlerOptions,
     BrowserCrawlingContext,
     BrowserHook,
+    PuppeteerCookie,
 } from '@crawlee/browser';
 import {
     Dictionary,
 } from '@crawlee/utils';
 import { BrowserPoolOptions, PuppeteerPlugin } from '@crawlee/browser-pool';
 import ow from 'ow';
 import { HTTPResponse, LaunchOptions, Page } from 'puppeteer';
+import { Cookie } from 'tough-cookie';
```

Comment:
undeclared dependency

Reply:
It comes from core, but I can explicitly add it too if needed 👍

Reply:
that's not how dependency resolution works, you are depending on an undefined behaviour that might differ across package managers (e.g. this would not work with yarn/berry with pnp enabled). dependencies you use need to be explicit.